roles/cockpit/tasks/main.yml

```diff
@@ -6,10 +6,11 @@
     - cockpit-shell
     - cockpit-bridge
     - "{{ cockpit_plugins }}"
-  when: not openshift.common.is_atomic | bool
+  when: not openshift.common.is_containerized | bool
```

Comment:
Any reason to switch to is_containerized here? The is_atomic check was put in specifically because we couldn't install the cockpit-kubernetes package on atomic.